models/DataObject/Concrete.php

```diff
@@ -323,7 +323,7 @@ public function saveVersion($setModificationDate = true, $saveOnlyVersion = true
     }
 
     /**
-     * @return array
+     * @return Model\Version[]
```

Comment:
could you change that for assets and documents as well ? :-)

Reply:
Give @weisswurstkanone a finger and he will grab the whole arm ;-)